lib/el_golang_lib/hook/fee_hook.go

```diff
@@ -19,21 +19,26 @@ const (
 	txSignatureOverhead = 68
 )
 
+var (
+	BaseFeeSlot  = common.BigToHash(big.NewInt(253))
```

Comment:
can you use the bindings in `ops` to get the slot? the bindings has the storage layout now